container-toolkit/install-guide.md

```diff
@@ -10,16 +10,16 @@
 
 ### Prerequisites
 
-Install the NVIDIA GPU driver for your Linux distribution.
+1. Install the NVIDIA GPU driver for your Linux distribution.
 NVIDIA recommends installing the driver by using the package manager for your distribution.
-
 For information about installing the driver with a package manager, refer to
 the [_NVIDIA Driver Installation Quickstart Guide_](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html).
+Alternatively, you can install the driver by [downloading](https://www.nvidia.com/en-us/drivers/) a `.run` installer.
+
 
-Alternatively, you can install the driver by downloading a `.run` installer.
-Refer to the NVIDIA [Official Drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) page.
+2. Read [this section](./supported-platforms.md) about platform support.
```

Comment:
Nit: Should this be first?

Reply:
+1 i think it should be first as well

and an additional nit to embed the text into the link, so something like 
``` Make sure you are using a [supported platform](./supported-platforms.md).```

Reply:
> Nit: Should this be first?

Yes, I was hoping for that feedback!

Reply:
>  something like Make sure you are using a [supported platform](./supported-platforms.md).

In general I like this approach. Here I opted for a subtly different approach because:

It's really a section about platform support in general, and we precisely now want to be mindful to _not_ communicate something in the lines of "do not continue if you do not use a strictly supported platform" :).

Related:   https://github.com/NVIDIA/cloud-native-docs/pull/62 and e.g. https://github.com/NVIDIA/nvidia-container-toolkit/issues/482#issuecomment-2122204585.